versions/1.0.0.md

```diff
@@ -8,19 +8,27 @@ This document is licensed under [The Apache License, Version 2.0](https://www.ap
 
 ## Introduction
 
-TBD
+The Overlay Specification is an extension or companion to the [[OpenAPI]] specification.
+An Overlay describes a set of changes to be applied or "overlaid" onto an existing OpenAPI description.
```

Comment:
Do we want to restrict Overlays to only operate on OpenAPI documents? For the most part the spec could easily be applied to any JSON or YAML doc. And in several places we seem to allow base documents of other types, e.g. target is described as
> URL to the target document (such as an [[OpenAPI]] document) ...

Reply:
The repo README.md says

> The Overlay specification defines a way of creating documents that contain information to be merged with an OpenAPI description at some later point in time, for the purpose of updating the OpenAPI description with additional information.

Given that with [RFC6902](https://datatracker.ietf.org/doc/html/rfc6902) and [RFC7386](https://datatracker.ietf.org/doc/html/rfc7386) there are already two document formats for modifying arbitrary JSON documents, I'd keep to the original scope.

@kevinswiber @lornajane?

Reply:
@mikekistler How about merging this PR and opening a dedicated issue for defining the scope of Overlay target documents?